extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -147,11 +155,17 @@ export class LiveListManager {
   fetchDocument_() {
     let url = this.url_;
     if (this.latestUpdateTime_ > 0) {
-      url = addParamToUrl(
-        url,
-        'amp_latest_update_time',
-        String(this.latestUpdateTime_)
-      );
+      const parameters = this.enrolledInAppendRandomExperiment_
```

Comment:
I think there's a race condition here but the poll interval probably masks it. Probably worth testing to see.